tests/run.go

```diff
@@ -447,3 +605,18 @@ func verifyMountsInfo(actual []MountJSON, want []MountJSON) {
 		}
 	}
 }
+
+// CGMode is the cgroups mode of the host system.
+// Ref. https://github.com/containerd/cgroups/blob/cc78c6c1e32dc5bde018d92999910fdace3cfa27/utils.go#L38-L50
```

Comment:
```suggestion
// We copy the struct from containerd/cgroups [1] instead of using it as a library
// because it only builds on linux,
// while we don't really need the functions that make it only build on linux
// (e.g., determine the cgroup version of the current host).
//
// [1] https://github.com/containerd/cgroups/blob/cc78c6c1e32dc5bde018d92999910fdace3cfa27/utils.go#L38-L50
```